app/views/assignments/_list.html.erb

```diff
@@ -1,67 +1,24 @@
-<% assignments = Assignment.where(is_hidden: false).includes(:pr_assignment).order(:id) %>
-
-<% if assignments.empty? %>
+<% if assessments.empty? %>
   <p><%= t('assignments.none') %></p>
```

Comment:
Perhaps this translation should be changed from  "There are currently no assignments." to something that is more inclusive of both grade entry forms and assignments, upcoming and past. Perhaps "No assessments to show"

Reply:
Nice catch, let's go with "No assessments to display."

Reply:
Should I change the non english translations to `UPDATE ME!` like the other places? I'm worried about leaving the translations in a state where they could be missed being different than english.

Reply:
Ah, this should actually be a new key (`assessments.none`) rather than modifying an existing key.